inscriptions/ISic000570.xml

```diff
@@ -185,14 +190,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
+                <ab>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>s</abbr><ex>acrum</ex></expan>
+                    <lb n="2"/><name><expan><abbr>P</abbr><ex>ublio</ex></expan></name> <g ref="#interpunct">·</g> Servili <g ref="#interpunct">·</g> Paphroditi
```

Comment:
publii, not publio I think? genitive rather than dative
and `<choice><corr>E</corr><sic>I</sic></choice>paphroditi` i think for the cognomen, looking at the apparatus in CIL

Also, we need `<name>` on each of Servili and Epaphroditi.